packages/api/src/routes/medical/patient.ts

```diff
@@ -126,6 +134,12 @@ router.put(
     const cxId = getCxIdOrFail(req);
     const id = getFromParamsOrFail("id", req);
     const facilityIdParam = getFrom("query").optional("facilityId", req);
+    const rerunPdOnNewDemographics = stringToBoolean(
+      getFrom("query").optional("rerunPdOnNewDemographics", req)
+    );
+    const augmentDemographics = stringToBoolean(
+      getFrom("query").optional("augmentDemographics", req)
+    );
```

Comment:
Why do we need `rerunPdOnNewDemographics` and `augmentDemographics` wouldnt just one suffice? Same for patient create

Reply:
I also think we could just have a single `augmentDemographics: boolean`:
1. Patient created/updated;
2. Execute PD;
3. If `augmentDemographics: true` and found new demographics during PD:
   1. Augment patient demographics w/ the demo from links;
   2. Schedule a new PD w/ `augmentDemographics: false` (go to #2);
4. If `augmentDemographics: false` or no new demo during PD:
   1. Execute the regular flow (trigger DQ if it was scheduled).

This ^ should support all permutations, no?

Reply:
@leite08 yes, this is what I was thinking if we want to clean things up, but let's discuss on exactly what we want. The "second" parameter was for if you wan to the the *initial* PD with augmented data. So essentially a way to create every increasing augmented data (i.e. for round 2, start with augmented data, and find *more* augmented data).

Reply:
Irrelevant now: flow has been updated to de-couple rerun and scheduling.